src/components/FreelancerCard.astro

```diff
@@ -0,0 +1,141 @@
+---
+import { Image } from 'astro:assets';
+import type { ImageMetadata } from 'astro';
+const { freelancer } = Astro.props;
+
+// Import social icons
+import { Icon } from 'astro-icon/components';
+
+import SecondaryCTA from "@components/ui/buttons/SecondaryCTA.astro";
+
+// Define the type for a skill
+type Skill = string;
+
+// Define a boolean to check if any certification is true
+const hasSprykerCertifications = freelancer.sprykerCertifications.backEndDeveloper || freelancer.sprykerCertifications.solutionArchitect;
+
+// Function to convert country code to flag emoji
+function getFlagEmoji(countryCode: string) {
```

Comment:
i would rather move it to https://github.com/spryker-community/CommerceQuest-website/blob/main/src/utils/utils.ts

Reply:
Done